mobile/src/events/CreateModifyEvent.tsx

```diff
@@ -35,6 +38,7 @@ export const repetitionValues = [
 
 type Props = {
   event?: Event;
+  pod?: Pod;
```

Comment:
Should the pod be optional here? The event was optional because it is possible that you are passing in an event that you want to modify instead of creating a new one, but shouldn't every user be in a pod (making the pod more or less a required param here)?